gallery/aistudio-insights/microsoft.machinelearningservices-workspaces.json

```diff
@@ -0,0 +1,21 @@
+{
+	"$schema": "https://raw.githubusercontent.com/microsoft/Application-Insights-Workbooks/master/schema/gallery.json",

```

Comment:
reminder, this one won't actually *work* until the workbooks team actually adds this aistudio-insights as an official supported type in code, and i'm still waiting on the details for that from mail/teams.  given CCOA's and safedeploy its probably closing in on 10 days to be available if it happens today?